src/scss/components/heroSlide.scss

```diff
@@ -0,0 +1,249 @@
+
+  *{
+    margin:0;
+    padding: 0;
+  }
+.Hero {
+  overflow: hidden;
+}
+.Hero{
+  height:100vh ;
+}
```

Comment:
you can merge this into a single rule declaration. also should use lower case to implement Sass's [nesting ](https://sass-lang.com/guide#topic-3) on the whole file